wiki/Gameplay/Game_modifier/Easy/en.md

```diff
@@ -11,8 +11,7 @@ tags:
 
 ![EZ mod icon](/wiki/shared/mods/EZ.png "Easy (EZ) mod icon")
 
-*For other uses, see [Easy (disambiguation)](/wiki/Disambiguation/Easy).*
-
+*For other uses, see [Easy (disambiguation)](/wiki/Disambiguation/Easy).*\
```

Comment:
not really related to PR but this one missed a colon, and then amusingly the same mistake was copied to the translations because they probably thought it was on purpose...